Assets/Scroll.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+/// <summary>
```

Comment:
The answer is apparently yes, this is a form of generating xml documents similar to javadocs.